workers/transactional_email_service/helpers.ts

```diff
@@ -12,3 +12,25 @@ export const createTags = (name: string, ...tagTuples: [string, string][]) => {
 
   return tags;
 };
+
+export const defaultInfo = {
+  community: {
+    from: {
+      name: "CommunityOS",
+      email: "contacto@communityos.io",
```

Comment:
¿Tal considerar usar subdominios para los emails?
Creo que ayudaría a no mezclar la reputación entre los distintos tipos de envíos, más cuando estamos enviando mayormente emails transaccionales.

Más info:
https://www.mailgun.com/blog/email/the-basics-of-email-subdomains/#chapter-3
https://www.mailmodo.com/guides/email-subdomain/